pontoon/teams/urls.py

```diff
@@ -9,6 +9,11 @@
         views.teams,
         name='pontoon.teams'),
 
+    # AJAX: Request locale to be added to Pontoon
+    url(r'^teams/requestlocale/$',
+        views.request_locales,
+        name='pontoon.teams.requestlocale'),
```

Comment:
`teams/request/` and `'pontoon.teams.request'` would be more consistent. You'd also need to rename the existing `'pontoon.teams.request'` to `'pontoon.teams.request.projects'`. And move both URL rules together.